### PR TITLE
feat: Update testnet with separate dev chain

### PR DESCRIPTION
**Pull request by ltfschoen**

### CODE REVIEW

File: .env [hunk 1]
```diff
@@ -0,0 +1,10 @@
+BOOT_NODE_LOCAL_1=QmWYmZrHFPkgX8PgMgUpHJsK6Q6vWbeVXrKhciunJdRvKZ
+# e.g. local or testnet_latest
+CHAIN_VERSION=local
+# e.g. 127.0.0.1 or testnet-harbour.datahighway.com
+ENDPOINT_DEVELOPMENT=127.0.0.1
+ENDPOINT_TESTNET=testnet-harbour.datahighway.com
+# Development or Production
+NODE_ENV=Development
+NODE_KEY=88dc3417d5058ec4b4503e0c12ea1a0a89be200fe98922423d4334014fa6b0ee
+SENTRY_NODE_TESTNET_1=QmVuryfE427VRqrqqXsGuWpwBk4g8mGXgYmnt3f1v6j78r
```
Comment (cgroeschel) [inline .env:10]: Do we want this none existing sentry node left as an example only as it does not exist?
Also the .env file should be ignored as per .gitignore and .env.sample only kept in the repository.

File: Dockerfile [hunk 2]
```diff
@@ -1,28 +1,35 @@
...
-  && cargo build --release \
+RUN apt-get update && apt-get install -y build-essential wget cmake pkg-config libssl-dev \
+    openssl git gcc clang libclang-dev curl vim unzip screen bash \
+    && curl https://getsubstrate.io -sSf | bash -s -- --fast \
+    && . ~/.cargo/env \
+    && wget -O - https://sh.rustup.rs | sh -s -- -y \
+    && PATH=$PATH:/root/.cargo/bin \
+    && rustup update stable \
+    && rustup update nightly \
+    && rustup toolchain install nightly-2020-02-17 \
```
Comment (cgroeschel) [inline Dockerfile:20]: the nightly toolchain needs to stay on this version from back in February still?

File: scripts/docker-entrypoint-node-1.sh [hunk 3]
```diff
@@ -1,17 +1,18 @@
...
   --keystore-path "/tmp/polkadot-chains/node-1/keys" \
-  --chain ../src/chain-definition-custom/chain_def_testnet_v0.1.0.json \
-  --name "Example Validator Node 1" \
+  --chain ../src/chain-definition-custom/chain_def_${CHAIN_VERSION}.json \
+  --name "${NODE_ENV} Validator Node 1" \
   --port 30333 \
   --ws-port 9944 \
   --rpc-port 9933 \
-  --telemetry-url ws://telemetry.polkadot.io:1024 \
+  --telemetry-url disabled \
```
Comment (cgroeschel) [inline scripts/docker-entrypoint-node-1.sh:16]: Any reason to turn off telemetry for all nodes?

File: Dockerfile (hunk 2 above)
Comment (ltfschoen) [reply Dockerfile:20]: yes, until we update the version of Substrate that we're using just to play it safe

File: scripts/docker-entrypoint-node-1.sh (hunk 3 above)
Comment (ltfschoen) [reply scripts/docker-entrypoint-node-1.sh]: i've changed it so all the validators appear in telemetry again

**Comment by ltfschoen** [comment]
@cgroeschel  if happy with changes let me know and we can merge

**Comment by ltfschoen** [comment]
yes i guess we can

**Comment by cgroeschel** [comment]
After reading up a bit more on pruning it turns out that for eg. an archive node, it should not be turned on. That's because for historical chain data all blocks are required. So guess pruning is optional (depending on node role) and should just be clearly explained in the documentation.

**Pull request merged**